lib/dat-worker-pool.rb

```diff
@@ -8,26 +8,23 @@
 
 class DatWorkerPool
 
-  attr_reader :logger, :spawned
-  attr_reader :queue
+  attr_reader :logger, :queue
   attr_reader :on_worker_error_callbacks
   attr_reader :on_worker_start_callbacks, :on_worker_shutdown_callbacks
   attr_reader :on_worker_sleep_callbacks, :on_worker_wakeup_callbacks
   attr_reader :before_work_callbacks, :after_work_callbacks
 
-  def initialize(min = 0, max = 1, debug = false, &do_work_proc)
-    @min_workers  = min
-    @max_workers  = max
-    @debug        = debug
-    @logger       = Logger.new(@debug)
-    @do_work_proc = do_work_proc
+  def initialize(number_of_workers = 1, debug = false, &do_work_proc)
+    @number_of_workers = number_of_workers
```

Comment:
@jcredding can we change the naming of the variable/usage to "num"?  Seems to flow better with the 'min/max' of the prev implementation.

So, `(num = 1` and `@num_workers = num`, etc, etc?  If you don't like this for some reason I am missing, I'm cool leaving as is - this just jumped out at me looking at the diff here.